src/test/java/io/strimzi/kafka/bridge/http/HttpBridgeTest.java

```diff
@@ -3486,4 +3488,30 @@ void openapiTest(VertxTestContext context) {
                 context.completeNow();
             });
     }
+
+    @Test
+    void createConsumerWithGeneratedName(VertxTestContext context) {
```

Comment:
We also need a test to check the instanceid when the bridgeid is not provided.

Reply:
After our offline discussion, let's leave this test for a future PR when the tests refactoring will be done. It's because we need to start the bridge with a different configuration, actually with no bridgei

Reply:
We agreed to wait till https://github.com/strimzi/strimzi-kafka-bridge/pull/294 is merged